### PR TITLE
Add Atelewo Spider

### PR DESCRIPTION
**Pull request by speedygamer12**
Add Atelewo Spider & Update .gitignore to make an exception for the Atelewo output file in data.

### CODE REVIEW

File: config/atelewo.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+sitemap_urls:
+  
+  - "http://atelewo.org/post-sitemap.xml"
+
+custom_settings:
+  ROBOTSTXT_OBEY: false
+  JOBDIR: "crawls/atelewo_spider"
+  RESTART_INDICATOR: "atelewoorg.restart"
```
Comment (theyorubayesian) [inline config/atelewo.yaml]: This `RESTART_INDICATOR` is also only needed when using the Selenium middleware. Can you take it out?

File: src/otelemuye/spiders/atelewo.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import logging
+
+from bs4 import BeautifulSoup
+from ._base import ArticleData, CustomSitemapSpider
```
Comment (theyorubayesian) [inline src/otelemuye/spiders/atelewo.py]: Generally, I prefer one import per line. Can we stick to that convention for this project?

Comment (theyorubayesian) [reply src/otelemuye/spiders/atelewo.py]: Also, add a space between Lines 3 & 4. To separate third-party packages from imports from our own local package.

**Comment by theyorubayesian** [comment]
Can you use a better name for your PR? `Dev` does not say a lot about what your PR is about

**Comment by theyorubayesian** [comment]
Also, update the README in `spiders/` to include your spider in the table of existing spiders.

File: config/atelewo.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  CONCURRENT_REQUESTS: 32
+  OUTPUT_FILE: "data/atelewoorg.jsonl"
+  ITEM_PIPELINES:
+    otelemuye.pipelines.JsonWriterPipeline: 300
+    
+
```
Comment (theyorubayesian) [inline config/atelewo.yaml]: Also, take out all unnecessary empty lines in this file?

**Comment by speedygamer12** [comment]
The new commit has implemented this

**Review by theyorubayesian — CHANGES_REQUESTED** [review]

File: config/atelewo.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+sitemap_urls:
+
```
Comment (theyorubayesian) [inline config/atelewo.yaml]: Remove empty space

File: src/otelemuye/spiders/atelewo.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
+import logging
+
+from bs4 import BeautifulSoup
+from ._base import ArticleData
```
Comment (theyorubayesian) [inline src/otelemuye/spiders/atelewo.py:5]: Leave space between 3 & 4

**Pull request merged**